src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -39,11 +54,10 @@ public User (@NotNull String username, @NotNull int status, @NotNull String user
     @Column(name = "USER_KEY", unique = true)
     private String userKey;
 
-    @NotNull
     @Column(name = "EXPIRATION_TIME")
     private LocalDateTime expirationTime;
 
-    @OneToOne(fetch = FetchType.LAZY)
+    @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
```

Comment:
po usunięciu tego musisz pilnować kolejności w jakiej dodajesz i zapisujesz obiekty bo ten parametr to zmieniał (np. podczas zapisywania usera zapiswyał w DB cart i odwrotnie, stąd errory)